### ISSUE TITLE
error @directus/sdk-js@6.0.1: The engine "node" is incompatible with this module. Expected version ">=8.0.0 <13.0.0". Got "13.7.0"

### ISSUE THREAD

**Issue by edoardomarsili** [directus/sdk-js#135]
# 🐞 Bug report

### Description
Not compatible with node 13.7.0 on Mac OS High Sierra.

## 🔬 Minimal Reproduction
```bash
yarn add v1.21.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @directus/sdk-js@6.0.1: The engine "node" is incompatible with this module. Expected version ">=8.0.0 <13.0.0". Got "13.7.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

## 🌍 Your Environment

**SDK Version:** ```v6.0.1```

**Anything else relevant?**

Package manager: Yarn
Pkg manager version: 1.21.1
Node version: v13.7.0
OS: OSX High Sierra 10.13.6

**Comment by Zilus** [comment]
Same issue with Mac OS Catalina:

` yarn add @directus/sdk-js
yarn add v1.21.1
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error @directus/sdk-js@6.0.1: The engine "node" is incompatible with this module. Expected version ">=8.0.0 <13.0.0". Got "13.7.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
`

**Comment by edoardomarsili** [comment]
Will try to digg bit more into it

**Comment by WoLfulus** [comment]
We'll remove the maximum version restriction on next release.

**Comment by rijkvanzanten** [comment]
Fixed in #136
